roles/satellite-clone/tasks/online_backup_restore.yml

```diff
@@ -0,0 +1,17 @@
+- name: drop foreman database
```

Comment:
Lines 1 to 11 here are 100% similar to restore_postgresql_migration.yml.  I recommend:

1. rename restore_postgresql_migration.yml to restore_postgresql_dump.yml
2. Use this new restore_postgresql_dump.yml for both rhel_migration and online_backup scenarios.